### DISCUSSION TITLE
I made a armv7 build for prisma-engines but failed

### DISCUSSION THREAD

**Discussion by ImBIOS** [prisma/prisma#18766] [Q&A] [answered]
https://github.com/ImBIOS/prisma-armv7-builds
Can anyone help me regarding error in the github action in this library?

**Comment by nurul3101** [comment]
Hello @ImBIOS 👋 

To confirm, you are talking about this error of `failed to run custom build command for openssl-sys v0.9.83`, right?

<img width="1870" alt="Screenshot 2023-04-17 at 11 56 14 AM" src="https://user-images.githubusercontent.com/13049676/232401859-1f579c7e-24bf-4c3e-a6b2-8bd7ff4b62f7.png">

**Reply by ImBIOS** [reply]
Yeah, that builds workflow. The cargo is always returning an error after 9-10 minutes.

**Reply by nurul3101** [reply]
Our engineering team will have a look as soon as possible and respond on the Prisma Engines Issue: https://github.com/prisma/prisma-engines/issues/3881

**Reply by nurul3101** [reply]
We can continue the conversation on the Prisma Engine GitHub Issue.

**Comment by janpio** [comment, marked as answer]
FYI: Also an issue at https://github.com/prisma/prisma-engines/issues/3881
